lizard_efcis/views.py

```diff
@@ -178,6 +178,9 @@ def filtered_opnames(self):
 
             opnames = opnames.filter(
                 locatie__meetnet__in=meetnetten)
+        if parameter_ids:
+            ids_list = json.loads(parameter_ids)
```

Comment:
Er zijn meerdere dingen met IDs. En 'list' zegt niet zoveel. Ik zou het `parameter_ids` noemen.

Het ding dat uit de query parameters komt, wat is dat dan? Een comma-separated string? Je gooit het in json. Is dat de netste manier? Encoded json in de URL is niet zo fraai. @gijs, wat zou jij zeggen? Een lijstje IDs met komma's ertussen in de URL lijkt me mooier.

Echt lekker expliciet is als je zoiets als dit dan hebt:

    comma_separated_parameters = self.request.....('parameters')
    parameter_ids = comma_separated_parameters.split(',')